jenkins-pipelines/oss/rolling-upgrade/rolling-upgrade-gce.jenkinsfile

```diff
@@ -9,6 +9,6 @@ rollingUpgradePipeline(
     linux_distro: 'ubuntu-focal',
     use_preinstalled_scylla: true,
     test_name: 'upgrade_test.UpgradeTest.test_rolling_upgrade',
-    test_config: '''["test-cases/upgrades/rolling-upgrade.yaml", "configurations/gce/n2-highmem-32.yaml"]''',
-    internode_compression: 'all',
+    test_config: '''["test-cases/upgrades/rolling-upgrade.yaml", "configurations/gce/n2-highmem-64.yaml"]''',
```

Comment:
- Please leave it as 32.
- Also, please change the name of it to "...-gce-image".